server/server.ts

```diff
@@ -57,6 +59,23 @@ if (config.DATABASE_URL) {
   postgres.connect();
 }
 
+let discordBot: DiscordBot;
```

Comment:
This should probably run in a separate service

Reply:
Actually, that might make the endpoint logic more complicated. But we may need to scale to multiple instances of the server soon, and I'm not sure if that will work if multiple processes are logging into the same Discord bot

Reply:
What do you think would be the best way to do the communication between the server processes and the discord process?

Reply:
But that might not be necessary because as far as I can tell it is possible to access the same bot through multiple different client instances using the same token.

Reply:
My preferred solution is usually to set up an HTTP server on the bot and have the web instances call it. But if you think it might be OK to scale out, then we can try that first

Reply:
I tried it out by starting the dev server multiple times. Which I guess comes close to having multiple instances of the same server.